conf/conf.d/default.conf

```diff
@@ -11,7 +11,7 @@ map $sent_http_content_type $expires {
 server {
   listen 8080;
   expires $expires;
-  add_header Content-Security-Policy "default-src 'self' accounts.google.com app.powerbi.com; script-src 'self' 'unsafe-inline' 'unsafe-eval' apis.google.com *.google-analytics.com www.gstatic.com; connect-src 'self' *.firecloud.org *.broadinstitute.org storage.googleapis.com *.google-analytics.com profile-dot-broad-shibboleth-prod.appspot.com; img-src 'self' data: *.google-analytics.com; style-src 'self' 'unsafe-inline' www.gstatic.com; base-uri 'self'; form-action 'self'; font-src 'self' fonts.gstatic.com;";
+  add_header Content-Security-Policy "default-src 'self' accounts.google.com app.powerbi.com; script-src 'self' 'unsafe-inline' 'unsafe-eval' apis.google.com *.google-analytics.com www.gstatic.com; connect-src 'self' *.firecloud.org *.broadinstitute.org storage.googleapis.com *.google-analytics.com *.googleapis.com profile-dot-broad-shibboleth-prod.appspot.com; img-src 'self' data: *.google-analytics.com; style-src 'self' 'unsafe-inline' www.gstatic.com; base-uri 'self'; form-action 'self'; font-src 'self' fonts.gstatic.com;";
```

Comment:
If you have `*.googleapis.com`, do you need to list `storage.googleapis.com`?

Reply:
No ... can condense them ... will fix